### ISSUE TITLE
YAMNet Pre-training Dataset

### ISSUE THREAD

**Issue by Kipsora** [tensorflow/models#10368]
AudioSet has two different training datasets the unbalanced version and the balanced version. The docs haven't discussed anything on the pre-training process. While I totally understand that the training code is not available yet, I am especially wondering the following questions:

1. Which dataset was used for pre-training YAMNet?
2. Which optimizers was used? What is the batch size and the learning rate?

I tried to train myself version of the YAMNet but using only the balanced version the loss won't converge. I also tried to train on the unbalanced version but the maximum mAP I can get is only `0.24` and the best dprime is `2.39`

**Comment by dpwe** [comment]
YAMNet is trained on the full AudioSet training set.  It was trained with a batch size of 128, a learning rate of 1e-5, and the Adam optimizer.  However, we use various other modifications to a standard training scheme: We compensate for class imbalance by scaling the objective error per example (unpublished) and we automatically label silence frames (unpublished). We also propagate labels up the hierarchy on training.  Your d-prime is better than our baseline (2.318), but your mAP is worse (we reached 0.309).  I'm not surprised by this amount of disagreement.
